libs/wingc/src/ast.rs

```diff
@@ -84,6 +84,8 @@ pub enum TypeAnnotation {
 	String,
 	Bool,
 	Duration,
+  MutJson,
```

Comment:
Maybe you need to use tabs instead of spaces? For some reason our rust code uses tabs... (@MarkMcCulloh, should we normalize this across our repo?

Reply:
😅 yea I noticed this when I pushed, I thought my VS code was smart enough to detect what was going on in the file.

Reply:
We are already forcing tabs if you run rustfmt (https://github.com/winglang/wing/blob/main/.rustfmt.toml)

Reply:
> Maybe you need to use tabs instead of spaces? For some reason our rust code uses tabs... (@MarkMcCulloh, should we normalize this across our repo?

@MarkMcCulloh ☝️